src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -233,4 +233,9 @@ public boolean hasCustomProjectile() {
 		};
 	}
 
+	@Override public List<MCItem> providedMCItems() {

```

Comment:
It would be cool if getIconForMCItem would return an egg only, not whole entity preview texture :D

Reply:
I can do this